modules/seedtagBidAdapter.js

```diff
@@ -1,74 +1,87 @@
-import { isArray, _map, triggerPixel } from '../src/utils.js';
-import { registerBidder } from '../src/adapters/bidderFactory.js'
-import { VIDEO, BANNER } from '../src/mediaTypes.js'
-
-const BIDDER_CODE = 'seedtag';
-const SEEDTAG_ALIAS = 'st';
-const SEEDTAG_SSP_ENDPOINT = 'https://s.seedtag.com/c/hb/bid';
-const SEEDTAG_SSP_ONTIMEOUT_ENDPOINT = 'https://s.seedtag.com/se/hb/timeout';
-const ALLOWED_PLACEMENTS = {
+import { isArray, _map, triggerPixel } from "../src/utils.js";
+import { registerBidder } from "../src/adapters/bidderFactory.js";
+import { VIDEO, BANNER } from "../src/mediaTypes.js";
+
+const BIDDER_CODE = "seedtag";
+const SEEDTAG_ALIAS = "st";
+const SEEDTAG_SSP_ENDPOINT = "https://s.seedtag.com/c/hb/bid";
+const SEEDTAG_SSP_ONTIMEOUT_ENDPOINT = "https://s.seedtag.com/se/hb/timeout";
+const ALLOWED_DISPLAY_PLACEMENTS = {
   inImage: true,
   inScreen: true,
   inArticle: true,
-  banner: true,
-  video: true
-}
+  inBanner: true,
+};
```

Comment:
those parameters are only for display, by default video don't use them because it only have one placement (instream) which is define in the mediatype

Reply:
As mentioned, let's still define the inStream placement for video :)

Reply:
ok is now fixed, video need `placement : "inStream"`  in the bidder params